src/index.html

```diff
@@ -190,13 +190,54 @@ <h4 class="search-error hidden">Your search didn't return any results.</h4>
           </fieldset>
 
         </section>
+        <h2 class="recipe-list-search-message"></h2>
+        <section class="recipe-list-content1 content1 "></section>
       </main>
 
+      <!--      favorites view  -->
+      <main class="favorites-view hidden">
+        <h1 class="recipe-list-title">My favorite recipes</h1>
+
+        <section class="search-wrapper">
+          <div class="search">
+            <label for="searchBarFavorites"><i class="far fa-search"></i></label>
+            <input type="text"
+                  class="search-bar"
+                  id="searchBarFavorites"
+                  name="search-bar"
+                  placeholder="Search by recipe name or ingredient"
+                  size="40"
+            >
+            <h4 class="search-error hidden">Your search didn't return any results.</h4>
+          </div>
+
+          <fieldset id="filter-dropdown-favorites" class="dropdown">
+            <select name="tags" id="tag-selector-favorites" class="dropdown__selector" size="9" multiple>
+              <option value="all" class="dropdown__selector--item">All</option>
+              <option value="breakfast" class="dropdown__selector--item">Breakfast</option>
+              <option value="brunch" class="dropdown__selector--item">Brunch</option>
+              <option value="appetizers" class="dropdown__selector--item">Appetizers</option>
+              <option value="side-dishes" class="dropdown__selector--item">Side dishes</option>
+              <option value="main-courses" class="dropdown__selector--item">Main courses</option>
+              <option value="desserts" class="dropdown__selector--item">Desserts</option>
+              <option value="healthy" class="dropdown__selector--item">Healthy</option>
+              <option value="vegetarian" class="dropdown__selector--item">Vegetarian</option>
+            </select>
+            <button id="goFavoritesButton" class="dropdown__button">Go</button>
+          </fieldset>
+
+        </section>
+        <!-- <h2 class="recipe-list-search-message favorites-search-message">Placeholder Search Results</h2> -->
+        <h2 class="favorites-list-search-message">Placeholder Search Results</h2>
+        <section class="recipe-list-content1 content1 "></section>
+      </main>
```

Comment:
This is making me so excited to get to React so we have components instead of a bunch of repeats.